wagit/headers/GitConfig.h

```diff
@@ -0,0 +1,31 @@
+//
+// Created by Ranveer Aggarwal on 28/08/20.
+//
+
+#include <filesystem>
+
+#define MINI_CASE_SENSITIVE
+#include "../includes/ini.h"
+
+#ifndef WAGIT_GITCONFIG_H
+#define WAGIT_GITCONFIG_H
+namespace WAGit
+{
+    namespace FS = std::filesystem;
+    using Path = FS::path;
+
+    class GitConfig
+    {
+        long version = 0;
```

Comment:
m_version? Since it's private. Is that a general thing or a Visio thing? 😛

Reply:
https://stackoverflow.com/a/4237476

Reply:
We should probably still follow something to keep the members (especially private members) separated from normal variables. Would improved readability.

Reply:
Okay, fixing it. Based on Google style guides.